gnovm/tests/imports.go

```diff
@@ -567,6 +567,53 @@ func testPackageInjector(store gno.Store, pn *gno.PackageNode) {
 				m.Context = ctx
 			},
 		)
+		pn.DefineNative("TestSetPrevRealm",
+			gno.Flds( // params
+				"", gno.AnyT(),
+			),
+			gno.Flds( // results
+			),
+			func(m *gno.Machine) {
+				arg0 := m.LastBlock().GetParams1().TV
+
+				switch arg0.T.String() {
+				case "std.Address":
+					// Set PrevRealm as an user
+					addr := arg0.GetString()
+					ctx := m.Context.(stdlibs.ExecContext)
+					ctx.OrigCaller = crypto.Bech32Address(addr)
+					m.Context = ctx
+
+				case "string":
+					// Set PrevRealm as a realm
+					realm := arg0.GetString()
+
+					// Set PkgPath
+					for i := m.NumFrames() - 1; i > 0; i-- {
+						fr := m.Frames[i]
+						if fr.LastPackage == nil || !fr.LastPackage.IsRealm() {
+							if i != 1 {
```

Comment:
Just curious, what is this situation where `i != 1`?

Reply:
While testing with `gno test` command, it returned `m.NumFrames() = 6` 

https://github.com/gnolang/gno/blob/f50f33de653bd660fe1a2ceae430fd7885e97e9c/gnovm/stdlibs/stdlibs.go#L318-L335


`PrevRealm` iterates all over `m.NumFrames()` and look for `fr.LastPackage`changes.
To satisfy changes, `NumFrame[i] // i != 1` needs dummy value